syntax = "proto3";

package game;

message ack_result {
  int32 ec = 1;                             // 错误码
  string msg = 2;                           // 错误提示语句
}

// 空的消息体
message msg_empty {}

// 通用Ack
message msg_common_ack {
  ack_result result = 1;
}

// 玩家数据
message user_info {
  int32 userid = 1;
  uint32 win_count = 2;
  uint32 lose_count = 3;
}

// 账号数据
message account_info {
  int32 userid = 1;
  string account = 2;
  string password  = 3;
}
